userspace/libsinsp/container.cpp

```diff
@@ -763,6 +815,40 @@ bool sinsp_container_manager::parse_docker(sinsp_container_info* container)
 		container->m_imageid = imgstr.substr(cpos + 1);
 	}
 
+	sinsp_utils::split_container_image(container->m_image,
+					   container->m_imagerepo,
+					   container->m_imagetag,
+					   container->m_imagedigest);
+
+	if(container->m_imagedigest.empty())
```

Comment:
Just double checking--it's not possible to combine this call to get_docker with the one at the beginning of parse_docker ? I'm guessing the formats differ?

Reply:
I kept them separate because if `container->m_image` already container a full qualified image name there's no need to query the images endpoint at all.
Also, note that when using libcurl, we're using HTTP1.1 pipelining so it will be almost the same as doing it in a single `get_docker` call.

Reply:
Ok, sounds good. Just making sure.